docs/Reference/Server.md

```diff
@@ -24,6 +24,7 @@ describes the properties available in that options object.
   - [`onProtoPoisoning`](#onprotopoisoning)
   - [`onConstructorPoisoning`](#onconstructorpoisoning)
   - [`logger`](#logger)
+  - [`createRequestLogMessage`](#createRequestLogMessage)
```

Comment:
I'm not particularly fond of this name. I think something like `customRequestLog` would be better.

Reply:
> I'm not particularly fond of this name. I think something like `customRequestLog` would be better.

No problem I like that too!  I'll make sure to edit the variable name.